### PR TITLE
[cyberarkpas] Upgrade ECS to 8.0.0

### PR DESCRIPTION
**Pull request by r00tu53r**
Automated PR.

Upgrades ECS to 8.0.0.

### CODE REVIEW

File: packages/cyberarkpas/data_stream/audit/sample_event.json [hunk 1]
```diff
@@ -1,38 +1,58 @@
...
+            "timestamp": "Mar 05 03:00:06"
         }
     },
+    "data_stream": {
+        "dataset": "cyberarkpas.audit",
+        "namespace": "ep",
+        "type": "logs"
+    },
+    "ecs": {
+        "version": "1.12.0"
```
Comment (efd6) [inline packages/cyberarkpas/data_stream/audit/sample_event.json:29]: 8.0.0

File: packages/fireeye/data_stream/nx/sample_event.json [hunk 2]
```diff
@@ -24,15 +23,15 @@
         "version": "1.12.0"
```
Comment (efd6) [inline packages/fireeye/data_stream/nx/sample_event.json:23]: 8.0.0

**Review by efd6 — CHANGES_REQUESTED** [review]

File: packages/cyberarkpas/data_stream/audit/sample_event.json (hunk 1 above)
Comment (r00tu53r) [reply packages/cyberarkpas/data_stream/audit/sample_event.json:29]: Fixed.

File: packages/fireeye/data_stream/nx/sample_event.json (hunk 2 above)
Comment (r00tu53r) [reply packages/fireeye/data_stream/nx/sample_event.json]: Due to merge issue it was in fireeye. Its been resolved.

**Review by marc-gr — APPROVED** [review]

**Review by efd6 — APPROVED** [review]

**Pull request merged**